src/card.css

```diff
@@ -0,0 +1,88 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
```

Comment:
Remove redundant default value

Reply:
`font-weight: 400;` is default value, so this line does nothing, i.e can be removed